### PR TITLE
Add support for EP_KAIROS

### PR DESCRIPTION
**Pull request by millerc-ep**
This is to give our Flidor board the ability to interface with our Kairos board.

We're also looking to get the CI artifacts from this PR.

@trowbridgec @maclobdell

### CODE REVIEW

**Comment by trowbridgec** [comment]
@0xc0170 Are you able to give us any insight into the build issues on this PR?

**Comment by 0xc0170** [comment]
```
01:28:55 Traceback (most recent call last):
01:28:55   File "tools/mbedcli_compile.py", line 26, in <module>
01:28:55     from package_release_files import package_release_files
01:28:55   File "C:\daplink_build\workspace\daplink_build_mbedcli_pr\daplink\tools\package_release_files.py", line 32, in <module>
01:28:55     import info
01:28:55   File "C:\daplink_build\workspace\daplink_build_mbedcli_pr\daplink\test\info.py", line 60
01:28:55     ('k20dx_ep_kairos_if'                           True,       0x0000,     "bin"       ),
01:28:55                                                        ^
01:28:55 SyntaxError: invalid syntax
```

Sorry it  took me so long.

File: test/info.py [hunk 1]
```diff
@@ -57,6 +57,7 @@
     ("k20dx_mimxrt1050_evk_hyper_if",               True,       0x8000,     "bin"       ),
     ("k20dx_mimxrt1050_evk_qspi_if",                True,       0x8000,     "bin"       ),
     ('k20dx_ep_agora_if',                           True,      	0x0000,     "bin"       ),
+    ('k20dx_ep_kairos_if'                           True,       0x0000,     "bin"       ),
```
Comment (0xc0170) [inline test/info.py:60]:
```suggestion
    ('k20dx_ep_kairos_if',                           True,       0x0000,     "bin"       ),
```

Comment (trowbridgec) [reply test/info.py:60]: @millerc-ep Can you make the suggested change?

**Comment by trowbridgec** [comment]
@0xc0170 Thank you!

**Comment by 0xc0170** [comment]
I applied the fix (hope its OK) , restarting the test

**Comment by trowbridgec** [comment]
@0xc0170 Thank you!

**Comment by 0xc0170** [comment]
The failures look related to CI target rather than to this PR, we will check

**Comment by trowbridgec** [comment]
@0xc0170 Can you re-run CI on this PR? Thanks!

**Comment by loverdeg** [comment]
This plug into Flidor?

**Comment by trowbridgec** [comment]
@loverdeg yessir! The `EP_KAIROS` target uses the NXP LPC55S69.

**Comment by trowbridgec** [comment]
@0xc0170 Did this fail CI again?

**Comment by millerc-ep** [comment]
@0xc0170 Hey there, just checking in. Can we see the CI log if possible?

**Review by 0xc0170 — APPROVED** [review]

**Comment by 0xc0170** [comment]
Build passed, test failed but they are unrelated.

**Comment by millerc-ep** [comment]
Hey @0xc0170 , can we get the build artifacts from this too? Sorry, I meant to ask you earlier. Thanks!

**Comment by millerc-ep** [comment]
@0xc0170 , just pinging you again to see if we can get the kairos build artifacts from this. Thank you!

**Pull request merged**